app/database_etl/postgres_tables_handler/table_generator.py

```diff
@@ -203,6 +204,24 @@ def create_country_df(dashboard_source_df, current_time):
     country_df = add_latlng_to_df("country", "country_name", country_df)
     country_df['country_iso3'] = country_df["country_name"].map(lambda a: get_country_code(a))
 
+    # Populate HRP status column
```

Comment:
Thanks! To be clear, even though these are in the Countries table, they will be exposed on a standard data request, right?

Reply:
It seems like the GBD region and WHO region are also attributes of a country. Is there a reason that these fields aren't also in the country table, or that we are dealing with these fields differently?

Reply:
Not sure, IMO they should be in the country table as well